storage/database/db_manager.go

```diff
@@ -168,6 +168,16 @@ type DBManager interface {
 	DeleteTrieNode(hash common.ExtHash)
 	WritePreimages(number uint64, preimages map[common.Hash][]byte)
 
+	// Trie pruning
+	ReadPruningEnabled() bool
+	WritePruningEnabled()
+	DeletePruningEnabled()
```

Comment:
A comment seems required that says this function must not be explicitly called. If not, the pruning process would not be available by https://github.com/klaytn/klaytn/pull/1871/files#diff-de2bd0674bf71cd6e3d92cb5621577831308e4cc990a15933e5e853f3358e4e0R277. Is it correct?

Reply:
See https://github.com/klaytn/klaytn/pull/1914.